src/pollyCLI/pollyintegration.cpp

```diff
@@ -302,18 +302,34 @@ QVariantMap PollyIntegration::getUserProjectFiles(QStringList ProjectIds){
 // PURPOSE:
 //    This function parses the output of "createproject" command run from Qtprocess..
 // Return project id for the new project..
-// CALLS TO: run_qt_process,get_run_id
+// CALLS TO: run_qt_process,parse_id
 //
 // CALLED FROM: external clients
 
 
 QString PollyIntegration::createProjectOnPolly(QString projectname){
     QString command2 = "createProject";
     QList<QByteArray> result_and_error = run_qt_process(command2, QStringList() << credFile<< projectname);
-    QString run_id = get_run_id(result_and_error.at(0));
+    QString run_id = parse_id(result_and_error.at(0));
     return run_id;
 }
 
+// name OF FUNCTION: createWorkflowRequest
+// PURPOSE:
+//    This function parses the output of "createWorkflowRequest" command run from Qtprocess..
+// Return workflow request id for the pollyphi workflow..
+// CALLS TO: run_qt_process,get_workflow_request_id
+//
+// CALLED FROM: external clients
+
+
+QString PollyIntegration::createWorkflowRequest(QString projectId){
+    QString command2 = "createWorkflowRequest";
```

Comment:
@Ky5595 Please use camel casing for variable names as well. Just for the code added in this PR. Let the rest be. I've changed those in my branch.